README.md

```diff
@@ -24,7 +24,7 @@
 - [Sponsor](#sponsor)
 
 ## Short Description
-An automation framework for rapidly and consistently deploying production-ready DLT platforms.
+An automation framework for rapidly and consistently deploying production-ready Distributed Ledger Transaction (DLT) platforms.
```

Comment:
Should it not be `Distributed Ledger Technology` ?

Reply:
Fixed